libs/product/src/drivers/magento/models/configurable-product.ts

```diff
@@ -0,0 +1,39 @@
+import { MagentoProduct } from './magento-product';
+import { MagentoSimpleProduct } from './simple-product';
+
+export interface MagentoConfigurableProduct extends MagentoProduct {
+	configurable_options: MagentoConfigurableProductOption[];
+	variants: MagentoConfigurableProductVariant[];
+}
+
+export interface MagentoConfigurableProductOption {
+	attribute_code:	string;
+	attribute_id:	string;
+	id:	number;
+	label: string;
+	position:	number;
```

Comment:
This is useful, we should come up with a `DaffSortable` interface that sortable features can rely on.